packages/inter-protocol/test/vaultFactory/vault-contract-wrapper.js

```diff
@@ -79,31 +80,41 @@ export async function start(zcf, privateArgs, baggage) {
     return floorDivideBy(getAmountOut(quoteAmount), LIQUIDATION_MARGIN);
   };
 
-  const reallocateWithFee = (fee, wanted, seat, ...otherSeats) => {
-    const toMint = AmountMath.add(wanted, fee);
+  /** @type {MintAndTransfer} */
+  const mintAndTransfer = (mintReceiver, toMint, fee, nonMintTransfers) => {
+    const kept = AmountMath.subtract(toMint, fee);
     runMint.mintGains(harden({ Minted: toMint }), stage);
+    /** @type {import('@agoric/zoe/src/contractSupport/atomicTransfer.js').TransferPart[]} */
+    const transfers = [
+      ...nonMintTransfers,
+      [stage, vaultFactorySeat, { Minted: fee }],
```

Comment:
Please rename `stage`, and drop the `stage.clear()` on line 97.  It's no longer being used for staging, and people familiar with stagings will be confused by this naming.

Are there other calls to `.clear()` in vaults or staking that can also be dropped?